core/controllers/custom_landing_pages_test.py

```diff
@@ -57,30 +57,3 @@ def test_invalid_topic_landing_page_leads_to_404(self):
     def test_valid_subject_and_topic_loads_correctly(self):
         response = self.get_html_response('/learn/maths/fractions')
         response.mustcontain('<topic-landing-page></topic-landing-page>')
-
-
-class StewardsLandingPageTest(test_utils.GenericTestBase):
```

Comment:
Why are you just deleting this? That's not right.

Reply:
This function looks for strings in the code. But since they are not loaded anymore, as directives are not loaded in backend tests. So it would just have `<stewards-landing-page></srewards-landing-age>`. Should I test for this then?

Reply:
Yes.

Reply:
I have added them. PTAL!